web_search_with_and/static/src/js/control_panel_model_extension.js

```diff
@@ -0,0 +1,56 @@
+odoo.define(
+    "web_search_with_and/static/src/js/control_panel_model_extension.js",
+    function (require) {
+        "use strict";
+
+        const {patch} = require("web.utils");
+        const components = {
+            ControlPanelModelExtension: require("web/static/src/js/control_panel/control_panel_model_extension.js"),
+        };
+
+        const patchObject = components.ControlPanelModelExtension.prototype;
```

Comment:
I had to extract this to a variable as pre-commit was generating a huge diff otherwise. Here only the `.prototype` part was added.